### DISCUSSION TITLE
Windows growing over time

### DISCUSSION THREAD

**Discussion by eightmeow** [FreeTubeApp/FreeTube#6147] [Q&A]
I size my windows so that they are just small enough that the main video takes the full width of the window. And then I make it a little smaller than that. This will work for a day, maybe a couple days, but eventually it will start sizing the windows larger than the size I set, and the layout of the window changes to allow thumbnails up the side. To be sure I wasn't imagining it, or there's not something else about the layout that is causing this, I screen-capped it immediately after setting it to the size I want, and a couple days later when it is no longer working I can compare, and I can confirm that the window is wider than it was when I set it up.

What is causing this?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Do you have multiple screens that both have different levels of scaling, e.g. one has 100% the other has 125%?

**Reply by eightmeow** [reply]
No. One screen, scaled 100%.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
One thing comes to mind. You can also adjust the scale by using the shortcuts `ctrl + +` or `ctrl + -` just like in the browser but when closing and reopening the application it goes back to normal scale

**Comment by abeber** [comment]
I have exactly the same issue - thought I was going mad for a while. Single screen only. Has been happening since I first used FT. So bug has never been addressed I think.
